_schemaV2.graphql

```diff
@@ -17502,6 +17514,26 @@ type SearchCriteriaEdge {
   node: SearchCriteria
 }
 
+enum SearchCriteriaFields {
+  acquireable
```

Comment:
**question:** You probably already thought of this, but I'm just noticing the duplication here of existing concepts such as [this](https://github.com/artsy/metaphysics/blob/31e3cd907280075aa67032fcc3dbaa658b316ac6/src/schema/v2/previewSavedSearch/previewSavedSearch.ts#L23) and [this](https://github.com/artsy/metaphysics/blob/31e3cd907280075aa67032fcc3dbaa658b316ac6/src/schema/v2/Alerts/index.ts#L72). Not sure if there's a way to derive this from those somehow?

Reply:
Thanks, good point, now I rely on `previewSavedSearchArgs`: https://github.com/artsy/metaphysics/pull/5573/files#diff-a1c3d020c591abdccdf06c556f53ea2c9ccf069bc8a523b318969d793b2e2b11R193

Maybe it's worth renaming and extracting to a common place..